changes/100.housekeeping

```diff
@@ -0,0 +1 @@
+Release 3.1.0 merge back into develop.
```

Comment:
I would prefer to leave these out of the release notes. It doesn't add any value.

Reply:
Also the changelog step doesn't block merging, specifically for this reason